Bottom: 52646e829b2c835e9171de316383f334d9b3e858
Top:    b3d747791c6e00f127bed74d0194e39b84d6c9fd
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-22 07:42:34 -0300

Refresh of wip-manual-document-mt-safety-locale.patch

---

diff --git a/manual/locale.texi b/manual/locale.texi
index fb6de4e..c61360b 100644
--- a/manual/locale.texi
+++ b/manual/locale.texi
@@ -243,6 +243,8 @@ The symbols in this section are defined in the header file @file{locale.h}.
 @c  _nl_find_locale
 @c   getenv LC_ALL and LANG envromt
 @c   _nl_load_locale_from_archive
+@c    sysconf _SC_PAGE_SIZE ok
+@c    
 @c   _nl_expand_alias asmalloc, selfdeadlock, memleak, fdleak, lockleak
 @c    libc_lock_lock selfdeadlock, lockleak
 @c    bsearch ok
